ansible.cfg

```diff
@@ -1,6 +1,6 @@
 [defaults]
 
-library            = /etc/ansible/ceph_plugins/library:/etc/ansible/ceph_roles/ceph-ansible/library
+library            = /etc/ansible/ceph_plugins/library:./library:/etc/ansible/ceph_roles/ceph-ansible/library
```

Comment:
I wonder if this path (since its not absolute) will break depending on which directory you use it from. That's probably minor since we can dictate where the playbooks are run from.

Reply:
Should a step be added to bootstrap the library into an alternate location that can be reliably sourced using the full path?  Or just leave it for now and allow that to be its own task if using the relative path becomes an problem?